src/lib/components/SelectGrid.svelte

```diff
@@ -4,29 +4,65 @@ A component to display menu options in a grid.
 -->
 <script lang="ts">
     import { createEventDispatcher } from 'svelte';
+    import { globalConfig } from '$lib/data/stores';
     export let options = [''];
-    export let cols = 5;
+    export let cols = 6;
+
     const dispatch = createEventDispatcher();
     $: rows = Math.ceil(options.length / cols);
 
+    let textColor = $globalConfig.themes
```

Comment:
Lots of lines just to get styling for the dropdown.

Reply:
Looks better, but could benefit from a themeing store or themeing defined elsewhere